lang/cs/Org.Apache.REEF.Examples.HelloREEF/HelloREEFYarn.cs

```diff
@@ -115,6 +114,17 @@ private void LogApplicationReport()
         }
 
         /// <summary>
+        /// Kill Job Application.
+        /// </summary>
+        /// <param name="appId"></param>
+        private void KillJobApplication(string appId)
```

Comment:
It is a bit confusing to have that method in a user example - and or C# version of HelloREEF is already quite long for a hello world application IMO. I would remove it from HelloREEF and write a separate unit test to check that kill actually works.